cohort_4/playbook/SCHEDULE.md

```diff
@@ -96,34 +106,52 @@ partners.
 | 9:20 AM  | Energizer                                                    |
 | 9:45 AM  | [Breakfast](ACTIVITIES.md#breakfast)                         |
 | 10:00 AM | [Project time](ACTIVITIES.md#freeform-projects)              |
+|          | [Scoping and concept](ACTIVITIES.md#scoping-and-concept)     |
+|          | [Engagement](ACTIVITIES.md#engagement)                       |
+|          | [Facilitating](ACTIVITIES.md#facilitating)                   |
+|          | [Answering questions](ACTIVITIES.md#answering-questions)     |
 | 12:30 PM | [Lunch](ACTIVITIES.md#lunch)                                 |
 | 1:30 PM  | Energizer                                                    |
 | 2:00 PM  | [Project time](ACTIVITIES.md#freeform-projects)              |
+|          | [Scoping and concept](ACTIVITIES.md#scoping-and-concept)     |
+|          | [Engagement](ACTIVITIES.md#engagement)                       |
+|          | [Facilitating](ACTIVITIES.md#facilitating)                   |
+|          | [Answering questions](ACTIVITIES.md#answering-questions)     |
```

Comment:
Same thing here. Does it make sense to link to each individual section from here?